src/main/java/org/mitre/stix/DocumentUtilities.java

```diff
@@ -35,21 +35,46 @@
 
 /**
  * A collection of utility helper methods.
- * 
+ *
  * @author nemonik (Michael Joseph Walsh <github.com@nemonik.com>)
  */
 public class DocumentUtilities {
 
 	private static final String XML_SCHEMA_INSTANCE = "http://www.w3.org/2001/XMLSchema-instance";
 	private static final String XML_NAMESPACE = "http://www.w3.org/2000/xmlns/";
 
+	public static JAXBContext stixJaxbContext() {
```

Comment:
I made this method public intentionally, because I lacked it in my project and find it useful for users who want more control over marshalling/unmarshalling.